src/components/Button/button.module.css

```diff
@@ -29,7 +28,9 @@
   --denhaag-button-medium-size-padding-inline: var(--denhaag-button-padding-inline);
   --denhaag-button-large-size-padding-block: 10px;
   --denhaag-button-large-size-padding-inline: 20px;
+}
```

Comment:
Ideally this `:root {}` declaration would be moved to the new theme package, and the button component itself does not come with any The Hague specific styles. Or to put it differently: component CSS files are best kept side-effect free, only affecting the page when any of the specified class names are applied to the HTML.

Reply:
This will be discussed with @mjcarsjens